src/Sylius/Bundle/UiBundle/Resources/views/Grid/Action/links.html.twig

```diff
@@ -1,16 +1,18 @@
-<div class="ui{% if options.icon is defined %} labeled icon{% endif %} floating dropdown {{ options.class|default('') }} link button">
-    {% if options.icon is defined %}<i class="{{ options.icon }} icon"></i>{% endif %}
-    <span class="text">{{ action.label|trans }}</span>
-    <div class="menu">
-        {% if options.header is defined %}
-        <div class="header">
-            {% if options.header.icon is defined %}<i class="{{ options.header.icon }} icon"></i>{% endif %}
-            {{ options.header.label|trans }}
+{% if options.visible|default(true) %}
```

Comment:
It is the feature of `links.html.twig` only, and we have declared this variable already [here](https://github.com/Sylius/Sylius/pull/7644/files#diff-49971219e0092c4c2fb3635aa788e80fR58) so I assume we are supporting it. This is also a clue of this PR